projects/overlook.md

```diff
@@ -143,76 +166,118 @@ password: overlook2021
+- Instructors are available to offer support and guidance but conversations around what is and what is not working are expected to be led by the team members themselves.
 
-Make sure you submit your project [here](https://docs.google.com/forms/d/e/1FAIpQLScsgrJD22g9WnUj7-3gXMHFSPqkk9rTt86kbRTEDGfGCIMLVA/viewform?usp=sf_link){:target='blank'} by **Tuesday of Week 6 at 12pm**.
+---
 
 # Rubric
 
-## Specification Adherence
+For the rubric sections below, you will be scored as Wow, Yes or Not Yet depending on whether you have demonstrated competency in that area. Each section lists examples of what types of things we may be looking for as demonstrations of competency. Just as there are many ways to approach code, there are many many ways to demonstate competency. The following examples are not checklists to complete! There are just some examples.
 
-* 4: The application completes all iterations above without error.
-* 3: The application completes the first 4 iterations above without error. **Note: Must be completed in order to pass.**
-* 2: The application completes the first 2 iterations and is in a usable state, but has some miscellaneous bugs.
-* 1: The application completes only the first iteration, displaying the user's data, but has no additional functionality.
+<section class="answer">
+### Functional Expectations
 
-## UI/UX & Accessibility
+- Wow: Application fulfills all requirements as well as an extension.
+- Yes: Application fulfills all requirements of iterations 1-3 without bugs. **Note: Must be completed in order to pass**
+- Not Yet: Application crashes or has missing functionality or bugs.
+</section>
 
-* 4: Application has clearly had special consideration around accessibility. Lighthouse accessibility audit is at a 100% and application is fully tabbable.
-* 3: Application has many strong pages/interactions. The application can stand on its own to be used by instructor without guidance. The UI does not detract from the UX. Lighthouse accessibility audit is at least 90% and application is fully tabbable.
-* 2: The application may be confusing or difficult to use at times.  The UI is incomplete. Accessibility has been considered, but does not have strong accessible features.
-* 1: Application is confusing or difficult to use. The UI is incomplete. Accessibility has not been considered.
+<section class="answer">
+### UI/UX & Accessibility
 
-## JavaScript Style & OOP
+* The application can stand on its own to be used by an instructor without guidance from a developer on the team.
+UI/UX is intuitive and easy to read/use
+* Helpful messaging is displayed to prevent user confusion. For example: For example: If a user searches for a room, but none are available then a message is displayed to indicate that the search worked, nothing is broken, there just aren't any matching rooms available.
+* The Lighthouse accessibility audit score is at least 90%
+* The application is fully tabbable 
```

Comment:
Do we want to add a competency example around aria tags since they're noted in the Accessibility section of the spec (maybe mentioning using them sparingly, only if neccesary)?

Reply:
Totally agree that its kind of weird since they usually dont need to be using Aria in the projects.  I love the clearer wording you outlined here for competency examples.  Honestly either Yes or Wow sounds good to me.  I'm on board for either.

> "Project utilizes ARIA attributes on interactive elements when no other tool allows for accessible content." and "ARIA attributes are not used when other tools allow for accessible content."

Reply:
Updated! Let me know if you'd prefer something else.